maven/pom.xml

```diff
@@ -98,17 +98,18 @@
 					<plugin>
 						<groupId>com.github.spotbugs</groupId>
 						<artifactId>spotbugs-maven-plugin</artifactId>
-						<version>3.1.3</version>
+						<version>3.1.12.2</version>
 						<dependencies>
 							<dependency>
 								<groupId>com.github.spotbugs</groupId>
 								<artifactId>spotbugs</artifactId>
-								<version>3.1.3</version>
+								<version>4.0.1</version>

```

Comment:
Do these Versions comply with the version used in our Eclipse IDE?

Reply:
Yes, assuming one has the newest built eclipse ide.